src/Data/Formatter/DateTime.purs

```diff
@@ -100,8 +104,10 @@ printFormatterF cb = case _ of
   Hours24 a → "HH" <> cb a
   Hours12 a → "hh" <> cb a
   Meridiem a → "a" <> cb a
-  Minutes a → "mm" <> cb a
-  Seconds a → "ss" <> cb a
+  Minutes a → "m" <> cb a
```

Comment:
I hear what you're saying, but unfortunately it is a breaking change, as the ADT has been modified and its constructors are exported.

I can make a release for 0.10 before updating to 0.11 though, if you need this and are sticking with 0.10 for now. Just it will have its major version bumped too :)